rust_src/src/dired_rust.rs

```diff
@@ -0,0 +1,572 @@
+//! Lisp functions for making directory listings.
```

Comment:
We use the name of the c file, so it should be dired.rs.

Reply:
Yea since the .c still exists I thought it might be a little safer (from .o clashes&such). But I'll rename it and see how it goes.